### PR TITLE
specify the db host

### PR DESCRIPTION
**Pull request by csmithATsquiz**
add option to specify the database host for the director to connect to (this is important for postgres where the default is to only allow "peer" connections and to use a password, you have to use "localhost" or an ip address).

Connecting bacula to postgres 9.5 without the host is giving this error:

```
 >FATAL:  Peer authentication failed for user "bacula"
 >DETAIL:  Connection matched pg_hba.conf line 11: "local	all	all		ident	"
```

because of these default pg_hba rules:
```
# This file is managed by Puppet. DO NOT EDIT.

# Rule Name: local access as postgres user
# Description: none
# Order: 001
local	all	postgres		ident	

# Rule Name: local access to database with same name
# Description: none
# Order: 002
local	all	all		ident	
```

### CODE REVIEW

**Comment by xaque208** [comment]
Cool, thank you for the pr.  This looks to be incomplete.  The variable needs to get used somewhere for this to work.

**Comment by csmithATsquiz** [comment]
Yeh, I forgot to commit the template change, oops.

Updated now :)

File: templates/bacula-dir-header.erb [hunk 1]
```diff
@@ -32,5 +32,6 @@ Catalog {
     Name   = MyCatalog
     dbname = "<%= @db_name %>";
     dbuser = "<%= @db_user %>";
-    dbpassword = "<%= @db_pw %>"
+    dbpassword = "<%= @db_pw %>";
+    DB Address = "<%= @db_host %>"
```
Comment (xaque208) [inline templates/bacula-dir-header.erb]: I believe this line will need a semi-colon at the end of it.

**Comment by xaque208** [comment]
Looking better.  This will need a rebase now that I've merged your other PR, and the semi-colon and this looks good to go.

**Comment by csmithATsquiz** [comment]
I'll close this off, tidy it up and re-submit.

**Pull request closed without merging**